### ISSUE TITLE
[Suggestion] TGA texture menu in Textures tab is missing some actions PNG has

### ISSUE THREAD

**Issue by PaulFinnerty** [JannisX11/blockbench#1787]
### Detailed description of your suggestion

Maybe there's a reason for the "Edit" type of tools, but I find "Show in File Explorer" to be missing in particular.

![image](https://user-images.githubusercontent.com/81629481/226097323-f073e335-7ac9-4353-b4d5-636d0de1b7b3.png)

**Comment by JannisX11** [comment]
Yeah Blockbench can only import TGA files, not export. So, when importing, the data is essentially imported without reference to the original TGA file, with the expectation to be exported as a regular PNG file later.

**Comment by BlueBatSamurai** [comment]
Could you please add support for saving a texture as a TGA file? 

When I made a Bedrock Edition resource pack with TGA textures, I had to save them as PNG and then convert them to TGA with Paint.NET. 

Why can you open a TGA file in Blockbench but then only save it as PNG?

**Comment by Cubouse** [comment]
@BlueBatSamurai  Please make your own Github issue to suggest this feature, even though your ask is closely related:

https://github.com/JannisX11/blockbench/issues/new/choose

Each and every issue or feature suggestion is kept separate, so that it is easier to track for the developers and other users: corresponding labels are given, people can upvote them individually, and the issues can be closed once implemented.

**Comment by BlueBatSamurai** [comment]
Hello @Cubouse.

Thanks for the tip!

I've now created an own suggestion issue for that:
https://github.com/JannisX11/blockbench/issues/2343
